packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -43,9 +48,9 @@ abstract class TickerProvider {
   Ticker createTicker(TickerCallback onTick);
 }
 
-// TODO(jacobr): make Ticker use Diagnosticable to simplify reporting errors
-// related to a ticker.
-/// Calls its callback once per animation frame.
+/// Calls its callback once per animation frame, when enabled.
+///
+/// To obtain a ticker, consider [TickerProvider].
```

Comment:
```suggestion
/// Calls its [TickerCallback] once per animation frame, when enabled.
```

The `TickerProvider` reference could be moved to a "See also" section at the bottom, for consistency with other docs.

Reply:
I think mentioning TickerProvider up at the top is helpful here because when someone's just trying to get an animation working and finding their way around the API, that's the reader who can most benefit from an early signpost helping them redirect toward what they're looking for.

Reply:
Fair point!

I still do think it'd be nice to link to the typedef here.

```suggestion
/// Calls its [TickerCallback] once per animation frame, when enabled.
///
/// To obtain a ticker, consider using a [TickerProvider].
```

And this is very pedantic, but I also feel that a sentence flows better when adding an "-ing" verb after "consider" (other alternatives: "consider the" or "consider who/what/when/where/how").

Reply:
For the summary line, I think the style I typically see in the Flutter docs tends to keep that summary line in plain English prose (no identifiers) except where an identifier reference is critical to understanding the meaning.

So for example [TickerCallback] itself says in its summary:

```dart
/// Signature for the callback passed to the [Ticker] class's constructor.
```

which fits because that type is really all about the `Ticker` class.

But here, I think the exact signature of the callback isn't critical to a summary-level understanding of what `Ticker` is about. So that detail is best left for later, past the summary. (In the [docs page](https://main-api.flutter.dev/flutter/scheduler/Ticker-class.html), it'll appear right after the class's doc comment, in the signature of the constructor.)

---

On "consider", I feel like I see the terser form a lot in the Flutter docs, and I think it works well when the object of it is short, like here.

Looking in the Git history confirms that that wording is often from Hixie (who wrote much of Flutter's documentation in general and set its style):
```
$ git log -p --author hixie -G 'consider\b'
    # or even, filtering to some of the most core docs:
$ git log -p --author hixie -G 'consider\b' lib/src/widgets/framework.dart
```

> consider the [AppLifecycleListener] API.

> consider a single [CustomPaint] widget.

> consider an expression such as the following: [… a one-line code block …]

> consider [SelectionArea].

and so on. Certainly plenty of "-ing" gerund forms too, but usually when there's more to say about it.